src/UseCases/AddDonation/AddDonationResponse.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\UseCases\AddDonation;
+
+use WMDE\Fundraising\Frontend\Domain\Model\Donation;
+use WMDE\Fundraising\Frontend\ResponseModel\ValidationResponse;
+
+/**
+ * @license GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class AddDonationResponse extends ValidationResponse {
```

Comment:
I'm not sure what we gain by extending `ValidationResponse`, as the constructor is made in a way that the response is always successful. Do you really need the functionality provided by `ValidationResponse`?